Dockerfile

```diff
@@ -5,18 +5,20 @@ WORKDIR ${HOME}
 # install Chrome
 RUN curl -sL http://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
 RUN echo 'deb http://dl.google.com/linux/chrome/deb/ stable main' >> /etc/apt/sources.list.d/google.list
-RUN apt-get update && apt-get install -y google-chrome-stable
+RUN apt-get update \
+    && apt-get install -y google-chrome-stable \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
`/var/cache/apt/archives/` may also contain the packages and could be cleared, but I've never seen it in Dockerfiles (may be Debian/Ubuntu-specific). Does that help with the size?

Reply:
(it's already empty in the image, no worries)